stubs/CoreGenericFunctions.phpstub

```diff
@@ -762,13 +762,26 @@ function str_ireplace($search, $replace, $subject, &$count = null) {}
 /**
  * @psalm-pure
  *
+ * @return ($input is non-empty-string ? non-empty-string : ($pad_length is positive-int ? non-empty-string: string))
+ *
  * @psalm-flow ($input, $pad_string) -> return
  */
 function str_pad(string $input, int $pad_length, $pad_string = '', int $pad_type = STR_PAD_RIGHT): string {}
 
 /**
  * @psalm-pure
  *
+ * @todo update $multiplier to be `0|positive-int`
```

Comment:
Did you left this on purpose?

Reply:
Yes. `str_repeat()` throws warnings when given negative int, but Psalm was emitting too many false positive coercion issues (int -> positive int) even on its own codebase.